app/services/preserved_object_handler.rb

```diff
@@ -91,14 +91,8 @@ def confirm_version
     if invalid?
       results << result_hash(INVALID_ARGUMENTS, errors.full_messages)
     else
-      Rails.logger.debug "confirm_version #{druid} called and object exists"
-      confirm_results = with_active_record_rescue do
-        po_db_object = PreservedObject.find_by!(druid: druid)
-        results.concat(confirm_version_on_db_object(po_db_object, :current_version))
-        pc_db_object = PreservedCopy.find_by!(preserved_object: po_db_object, endpoint: endpoint)
-        results.concat(confirm_version_on_db_object(pc_db_object, :version))
-      end
-      results.concat(confirm_results)
+      Rails.logger.debug "confirm_version #{druid} called"
```

Comment:
why remove the specificity of the debug logging messages when they are still in the same different conditions?